examples/2016-10-31/api_cognito_auth/template.yaml

```diff
@@ -0,0 +1,153 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: API Gateway + Cognito User Pools Auth
+
+Parameters:
+  CognitoUserPoolName:
+    Type: String
+    Default: MyUserPool
+  
+  CognitoUserPoolClientName:
+    Type: String
+    Default: MyUserPoolClient
+
+Resources:
+  MyApi:
+    Type: AWS::Serverless::Api
+    Properties:
+      StageName: Prod
+      Cors: "'*'"
```

Comment:
This seems wrong to me. It is a SAM thing that we want it to be a string?

Reply:
SAM CORS uses API Gateway response mapping, and this is an implementation detail from API Gateway.